cupy/cuda/stream.pyx

```diff
@@ -470,6 +472,9 @@ class Stream(_BaseStream):
                                  'default stream (ptds)')
             ptr = runtime.streamPerThread
             device_id = -1
+        elif priority:
+            ptr = runtime.streamCreateWithPriority(runtime.streamNonBlocking, priority)
```

Comment:
`non_blocking` may be False so we need to check the value for `flags` parameter.

Reply:
Thank you! I've thought about this but ended up going with the other version for some reason. Fixed now~